### PR TITLE
add bird meme

### LINKED ISSUE #257: some cat meme

### PR DESCRIPTION
**Pull request by grace1501**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,5 +130,6 @@ Clone this repository (or open in codespace). Create an issue with a meme descri
...
-| ![](https://scontent-ord5-2.xx.fbcdn.net/v/t1.6435-9/75521867_1748245848640690_8667252245370241024_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0b6b33&_nc_ohc=SiocJFF-QxAQ7kNvgF9yHUi&_nc_zt=23&_nc_ht=scontent-ord5-2.xx&_nc_gid=AWnERHsTW1bsQX56UF4KitG&oh=00_AYB3n9-vGBL9OVp-ht91AZ_10RhznhDHqD24umrYt7niXA&oe=6766068C) | [Max Montesino](https://github.com/m-nt-s-no) | [source](https://www.facebook.com/photo.php?fbid=1748245845307357&id=298385066960116&set=a.298401790291777) |
+| ![](https://scontent-ord5-2.xx.fbcdn.net/v/t1.6435-9/75521867_1748245848640690_8667252245370241024_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0b6b33&_nc_ohc=SiocJFF-QxAQ7kNvgF9yHUi&_nc_zt=23&_nc_ht=scontent-ord5-2.xx&_nc_gid=AWnERHsTW1bsQX56UF4KitG&oh=00_AYB3n9-vGBL9OVp-ht91AZ_10RhznhDHqD24umrYt7niXA&oe=6766068C) | [Max Montesino](https://github.com/m-nt-s-no) | [source](https://www.facebook.com/photo.php?fbid=1748245845307357&id=298385066960116&set=a.298401790291777) 
+| ![](/assets/bird-meme.jpeg) | [KieuThanhDiem Grace Do](https://github.com/grace1501) | [source](https://x.com/TommySiegel/status/1857805646755750000)
```
Comment (heratyian) [inline README.md]: Can remove leading slash `/`

**Review by heratyian — CHANGES_REQUESTED** [review]

**Review by marywebby — CHANGES_REQUESTED** [review]
There some merge conflicts you need to fix! you do not have the latest version of main

File: README.md [hunk 2]
```diff
@@ -130,6 +130,8 @@ Clone this repository (or open in codespace). Create an issue with a meme descri
...
+| ![](https://scontent-ord5-2.xx.fbcdn.net/v/t1.6435-9/75521867_1748245848640690_8667252245370241024_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0b6b33&_nc_ohc=SiocJFF-QxAQ7kNvgF9yHUi&_nc_zt=23&_nc_ht=scontent-ord5-2.xx&_nc_gid=AWnERHsTW1bsQX56UF4KitG&oh=00_AYB3n9-vGBL9OVp-ht91AZ_10RhznhDHqD24umrYt7niXA&oe=6766068C) | [Max Montesino](https://github.com/m-nt-s-no) | [source](https://www.facebook.com/photo.php?fbid=1748245845307357&id=298385066960116&set=a.298401790291777) 
+| ![](assets/bird-meme.jpeg) | [KieuThanhDiem Grace Do](https://github.com/grace1501) | [source](https://x.com/TommySiegel/status/1857805646755750000)
 | ![](https://scontent-ord5-2.xx.fbcdn.net/v/t1.6435-9/75521867_1748245848640690_8667252245370241024_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0b6b33&_nc_ohc=SiocJFF-QxAQ7kNvgF9yHUi&_nc_zt=23&_nc_ht=scontent-ord5-2.xx&_nc_gid=AWnERHsTW1bsQX56UF4KitG&oh=00_AYB3n9-vGBL9OVp-ht91AZ_10RhznhDHqD24umrYt7niXA&oe=6766068C) | [Max Montesino](https://github.com/m-nt-s-no) | [source](https://www.facebook.com/photo.php?fbid=1748245845307357&id=298385066960116&set=a.298401790291777) |
```
Comment (marywebby) [inline README.md:134]: I think Max is in here twice on line 133 and line 135! make sure hes on here once

**Review by marywebby — CHANGES_REQUESTED** [review]

**Review by marywebby — APPROVED** [review]
Looks great! Thank you for making those changes!

**Review by heratyian — APPROVED** [review]
Looks good! 😊

**Pull request merged**